sdk/python/feast/expediagroup/provider/expedia.py

```diff
@@ -68,8 +68,24 @@ def update_infra(
         entities_to_delete: Sequence[Entity],
         entities_to_keep: Sequence[Entity],
         partial: bool,
+        materialization_update: bool = False,
```

Comment:
This is an inherited method. If you need to modify this, you need to modify many changes. You can do this differently. 

We can get rid of this function in Expedia provider. Previously its added to override the update_infra behavior for redis online store. Now feast has config `full_scan_for_deletion`. You can default this to False in repo_config.py for Expedia provider. So all users using Expedia provider, will get the same behavior. Feast is discouraging providers. Going forward they may remove them as well. 

For scylladb, we can let the users set `lazy_table_creation` in their configuration `feature_store.yaml`.